__tests__/app.test.js

```diff
@@ -756,6 +756,81 @@ describe('CORE: GET /api/users', () => {
     })
 })
 
+describe('FEATURE: GET /api/articles (topic query)', () => {
+    test('Responds with all articles when topic is omitted in the query', () => {
+        const arrOfAllArticles = require('../db/data/test-data/articles_array');
+        arrOfAllArticles.sort((curr,next) => curr.article_id - next.article_id);
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then((response) => {
+            const articles = response.body.articles;
+            articles.sort((curr,next) => curr.article_id - next.article_id);
+            expect(articles.length).toBe(arrOfAllArticles.length);
+            expect(articles).toEqual(arrOfAllArticles);
+        })
+    })
+    test('Responds with all articles when the topic query is misspelt', () => {
```

Comment:
This feedback isn't going to be useful, But I have been staring at this for ages wondering if I agree with sending a 200 back in this case.